### ISSUE TITLE
Mosca not considering a Mosquitto bridge as a bridge and still delivering messages twice to it.. 

### ISSUE THREAD

**Issue by arvindsesha** [mcollina/mosca#343]
Not sure if this is the right Forum to post this issue, but anyways posting it to find some answers

I have a strange issue with a Mosquitto bridge on my android device connecting to Mosca on cloud.

The scenario is that I have a MQTT client connected to the Mosquitto bridge on the Android device and another client directly connected to the Mosca broker. When a message published through the mosca broker, the client directly connected to Mosca receives it only once [which is correct], but the client connected through the mosquitto bridge receives the same msg twice. The QOS is set to 1 at the bridge configuration and at the clients.

The mosquitto bridge configuration is as below

connection bridgetocloud
address Mosca-balancer-23232322323..elb.amazonaws.com:1883
topic mqtt/2/events/# both 1 
bridge_attempt_unsubscribe false
keepalive_interval 30
start_type automatic
restart_timeout 1
try_private true

We also noticed that when the android Mosquitto is bridged with a Mosquitto set up on one of our servers, the msg delivery behaves correctly with only only one msg recieved at the MQTT client.

Is mosca not considering the Bridge as a "bridge" and just as another client ?

**Comment by mcollina** [comment]
The Mosquitto bridge is not part of the formal specification. I would love to add that support, but I have no time. I would accept a PR on the matter.

This was already described in https://github.com/mcollina/mosca/issues/25

**Comment by arvindsesha** [comment]
If I need to add this support where should i start looking into.   I browsed through the code at a superficial level but couldnt find any support for bridging.  Any help and directions would be helpful.

**Comment by mcollina** [comment]
I believe the starting point should be MQTT.js. First we need a bridge
client that we know works with Mosquitto, and then we add support in here.
Il giorno lun 14 set 2015 alle 13:50 arvindsesha <notifications@github.com>
ha scritto:
